Packs/Cisco_OpenDNS/Integrations/CiscoOpenDNS/CiscoOpenDNS.py

```diff
@@ -0,0 +1,198 @@
+            status = False
+
+        #print("Result :", r2_j)
+        #print("Response Code :", r_code)
+
+        human_readable_data = {
+            'Domain': str(domain),
+            'Action Result Code': r_code,
+            'Response :': r2_j['id'],
+            'ActionResult': ActionResult
+        }
+
+        outputs = {
+            'OpenDNSblockDomain': {
+                'Domain': str(domain),
+                'Action Result Code': r_code,
+                'Response :': r2_j['id'],
+                'ActionResult': ActionResult
+            }
+        }
+
+        headers = ['Domain', 'Action Result Code', 'Response', 'ActionResult']
+        human_readable = tableToMarkdown('OpenDNSblockDomain info', human_readable_data, headers=headers, removeNull=True)
+        return_outputs(human_readable, outputs, r2_j)
+        return status
+    else:
+        human_readable_data = {
+            'Domain': str(domain),
+            'Action Result Code': "Already Blocked",
+            'Response :': "Already Blocked",
+            'ActionResult': "Already Blocked"
+        }
+
+        outputs = {
+            'OpenDNSblockDomain': {
+                'Domain': str(domain),
+                'Action Result Code': "Already Blocked",
+                'Response :': "Already Blocked",
+                'ActionResult': "Already Blocked"
+            }
+        }
+
+        headers = ['Domain', 'Action Result Code', 'Response', 'ActionResult']
+        human_readable = tableToMarkdown('OpenDNSblockDomain info', human_readable_data, headers=headers, removeNull=True)
+        return_outputs(human_readable, outputs)
+        return True
+
+
+def OpenDNSdeleteDomain(myDomainsrequest, myEventsrequest, domain):
+    #print("we are inside del function")
+    delRequest = myDomainsrequest + "&where[name]=" + str(domain)
+    #print("delRequest ",delRequest)
+    r3 = requests.delete(delRequest, timeout=2.50)
+    # r3_txt=r3.json()
+    #print("Response  ", r3_txt)
+    r_code = r3.status_code
+    print("delRequest status ", r_code)
+    if int(r_code) == 204:
+        print(domain + " Domain was removed from blacklist")
+    else:
+        print(domain + " Not in the blacklist or Error")
+
+
+def test_module(myDomainsrequest):
+    try:
+        r = requests.get(myDomainsrequest)
+        r_code = r.status_code
+        if int(r_code) == 200:
+            status = 'ok'
+        else:
+            status = 'Error'
+        # print(r_code)
+    except Exception as e:
+        LOG(e)
+        return_error(e.message)
+    demisto.results(status)
+
+
+def main():
+
+    domain = demisto.args().get('domain')
+    baseurl = demisto.params().get('url')
+    APIKey = demisto.params().get('APIKey')
+    myEventsrequest = str(baseurl) + "events?customerKey=" + str(APIKey)
+    myDomainsrequest = str(baseurl) + "domains?customerKey=" + str(APIKey)
+    # print(myDomainsrequest)
+    try:
+        if demisto.command() == 'OpenDNSblockDomain':
```

Comment:
Please change command names to fit our command standards (i.e. `open-dns-block-domain`). More information can be found [here](https://xsoar.pan.dev/docs/integrations/yaml-file#commands).
Don't forget to change the names in the `.yml` file as well.